### ISSUE TITLE
WCAG: That a link take you out of the app (external) are only communicated trough an icon

### ISSUE THREAD

**Issue by Febakke** [Altinn/app-frontend-react#231]
### Description of the bug

When using links that go out of the app it is common to have an arrow icon on the end of the link to indicate that you are leaving the page you are currently on if you use the link. But this information should also be communicated to screen readers.

This is handled on Altinn.no today and should be handled in similar way for Altinn 3 apps. 

[Example from Altinn.no:](https://www.altinn.no/om-altinn/hva-er-altinn/)
![Skjermbilde 2022-06-17 kl  12 12 30](https://user-images.githubusercontent.com/33222679/174279649-232cbbdf-6a57-4587-916b-b741e61bda2d.png)
 

### Steps To Reproduce

Use a screen reader on an external link in an application
There should be no indication that you are leaving the area you are in.

**Comment by Febakke** [comment]
Could no recreate this bug. Putting it in the analyze epic to double check if this is a bug.

**Comment by olemartinorg** [comment]
There was a problem where the arrow icon did not display correctly when using a local app-frontend (using the docker image or `yarn start`), but that is fixed in #893. With that said, we should decide if we want this arrow to show up on external links (or links that open in a new window, or both, or never) - as also discussed in #893.
